gen3-integration-tests/conftest.py

```diff
@@ -61,6 +62,15 @@ def pytest_collection_finish(session):
                     setup.get_fence_client_info()
                     requires_fence_client_marker_present = True
                     return
+                if marker_name == "requires_usersync":
+                    requires_usersync_marker_present = True
+                    return
+
+
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
let us not use autouse which will do it for all tests, instead import the fixture in required tests